src/components/Capacity.svelte

```diff
@@ -54,25 +39,17 @@
 
   let capacityDetails: CapacityDetails = defaultDetails;
 
-  transactionSigningAddress.subscribe(async (addr) => {
-    // first set/reset all our local values.
-    signingAddress = addr;
-    msaInfo = { isProvider: false, msaId: 0, providerName: '' };
+  user.subscribe(async (val) => {
     if (connected && apiPromise) {
       blockNumber = await getBlockNumber(apiPromise);
-      storeBlockNumber.update((val) => (val = blockNumber));
+      $storeBlockNumber = blockNumber;
```

Comment:
NOT this PR, but it feels off that the block number is being updated in the Capacity component for the entire app.

Reply:
I agree.  We'll address in a different PR.  File an issue?

Reply:
Yeah, add an issue, but I don't think we should do it until we are completely done with this set of work.